map/backup/doc/prototype.backup.md

```diff
@@ -4,12 +4,24 @@
 
 @description Save the values of the properties of an Map.
 
-@signature `map.backup()`
+@signature `map.backup( [options] )`
 
 `backup` backs up the current state of the properties of an Observe and marks
 the Observe as clean. If any of the properties change value, the original
 values can be restored with [can.Map.backup.prototype.restore restore].
 
+By default `backup` uses `attr` function on the map to get value backup value, 
+if you want to use `serialize` or custom method use `options` parameter 
+
+@param {object|string|bool} backup options:
```

Comment:
I diti it as in https://github.com/bitovi/canjs/blob/master/map/backup/doc/prototype.restore.md, if it is needed I can change there too for this commit.